app/src/main/kotlin/com/wire/android/ui/debug/DebugDataOptions.kt

```diff
@@ -124,6 +139,15 @@ class DebugDataOptionsViewModel
         }
     }
 
+    fun checkDependenciesVersion() {
+        viewModelScope.launch {
+            val dependencies = context.getDependenciesVersion().toImmutableMap()
+            state = state.copy(
+                dependencies = dependencies
```

Comment:
Can you run it on IO dispatcher ?

Reply:
i did change the function Context.getDependenciesVersion() to use IO